src/vorta/utils.py

```diff
@@ -245,6 +248,25 @@ def open_app_at_startup(enabled=True):
                                                  None, None, url, props, None)
         if not enabled:
             LSSharedFileListItemRemove(login_items, new_item)
+    elif sys.platform == 'linux':
```

Comment:
Will this also work on non-Gnome desktops and window managers? For Linux there is more fragmentation, so I'm generally skeptical.

Reply:
I just looked it up, and apparently it is recommended to use 'sys.platform.startswith('linux')' instead to maintain compatibility for python versions <3.3. I changed that now.

Otherwise it should work, but maybe @ThomasWaldmann and @antlarr can help to test it on xfce and KDE?